roles/update/templates/workload_launch_k8s.sh.j2

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+cat "{{ cifmw_update_artifacts_basedir }}/workload_launch.sh" | \
+    oc rsh -n openstack openstackclient env WKL_MODE=sanityfast bash
```

Comment:
```suggestion

export KUBECONFIG="{{ cifmw_openshift_kubeconfig }}"
export PATH="{{ cifmw_path }}"

cat "{{ cifmw_update_artifacts_basedir }}/workload_launch.sh" | \
    oc rsh -n openstack openstackclient env WKL_MODE=sanityfast bash
```
If we donot export kubeconfig and cifmw_path then we might hit following errors:
- oc binary not found
Below is the second error:
```
error: Missing or incomplete configuration info.  Please point to an existing, complete config file:


  1. Via the command-line flag --kubeconfig
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

To view or setup config directly use the 'config' command.
```

Reply:
Ok, added to the script. Thanks.